json-schema/schema.json

```diff
@@ -160,46 +162,24 @@
     "require_any_field": {
       "$comment": "Please list all fields here so that we can force the existence of one of them in other parts of the schemas.",
       "anyOf": [
-        {"required": ["template:new_field"]},
-        {"required": ["template:xyz"]},
-        {"required": ["template:another_one"]}
+        {"required": ["marble:host_node"]},
```

Comment:
Does this mean that either host_node OR is_local is required? Shouldn't both be required at the same time?

Reply:
No, it doesn't mean that. Both are required and extension in the present form will complain if either one is not present.